az-core/src/main/java/azkaban/Constants.java

```diff
@@ -89,6 +89,8 @@
   // The flow exec id for a flow trigger instance which hasn't started a flow yet
   public static final int UNASSIGNED_EXEC_ID = -1;
 
+  // The URL setting in H2 database to create tables with case insensitive texts.
+  public static final String H2_DB_CASE_INSENSITIVE = ";IGNORECASE=TRUE";
```

Comment:
I don't think this constant should belong to this class.

This class is designed to be one place to capture user facing config keys, until we find a better way... 

This constant can stay in the class that uses it.

Reply:
Sure, moved the constant back to original class.